package.json

```
@@ -159,6 +159,9 @@
             "Earth tones",
             "Everglade green",
             "Buttered toast",
+            "Terminal",
+            "Terminal Grayscale",
+            "Origami",
```

```suggestion
            "Origami",
            "Dark Flagship Terrastruct",
```

but below "Dark mauve"